RELEASE.md

```diff
@@ -29,6 +29,7 @@ Our goal is to provide a new minor release every 6 weeks. This is a new process
 | v1.11.0        | 2021-08-06                                 | Marco Pracucci (@pracucci)                  |
 | v1.12.0        | 2022-02-17                                 | _Abandoned_                                 |
 | v1.13.0        | 2022-06-23                                 | Alvin Lin (@alvinlin123)                    |
+| v1.14.0        | 2022-12-01                                 | Alan Protasio (@alanprot)                   |
```

Comment:
I think the date should be either today or tomorrow; it's the date for pre-release, which typically happens on the same day of this PR.